packages/stripe_platform_interface/lib/src/method_channel_stripe.dart

```diff
@@ -78,22 +79,22 @@ class MethodChannelStripe extends StripePlatform {
     PaymentMethodParams params, [
     Map<String, String> options = const {},
   ]) async {
-    try {
-      final result = await _methodChannel
-          .invokeMapMethod<String, dynamic>('confirmPaymentMethod', {
-        'paymentIntentClientSecret': paymentIntentClientSecret,
-        'params': params.toJson(),
-        'options': options,
-      });
-
-      final tmp = result?['paymentIntent'] as Map<String, dynamic>;
-
-      return PaymentIntent.fromJson(tmp.unfoldToNonNull());
-    } on Exception catch (_) {
+    final result = await _methodChannel
+        .invokeMapMethod<String, dynamic>('confirmPaymentMethod', {
+      'paymentIntentClientSecret': paymentIntentClientSecret,
+      'params': params.toJson(),
+      'options': options,
+    });
+    if (result == null) {
```

Comment:
can this ever happen? I don't think we should handle this specifically

Reply:
it is very theoretical. It doesn't hurt :)